test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/KeepAliveTimeoutTests.cs

```diff
@@ -19,7 +19,8 @@ public class KeepAliveTimeoutTests
         private static readonly TimeSpan LongDelay = TimeSpan.FromSeconds(30);
         private static readonly TimeSpan ShortDelay = TimeSpan.FromSeconds(LongDelay.TotalSeconds / 10);
 
-        [Fact]
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, SkipReason = "https://github.com/aspnet/KestrelHttpServer/issues/1684")]
```

Comment:
Looks like you need to add a using. https://ci.appveyor.com/project/aspnetci/kestrelhttpserver/build/1.0.5298#L253